src/main/java/it/gov/pagopa/payhub/activities/activity/paymentsreporting/PaymentsReportingIngestionFlowFileActivityImpl.java

```diff
@@ -52,7 +61,12 @@ public PaymentsReportingIngestionFlowFileActivityResult processFile(Long ingesti
 			CtFlussoRiversamento ctFlussoRiversamento = flussoRiversamentoUnmarshallerService.unmarshal(ingestionFlowFile);
 			log.debug("file CtFlussoRiversamento with Id {} parsed successfully ", ctFlussoRiversamento.getIdentificativoFlusso());
 
-			return new PaymentsReportingIngestionFlowFileActivityResult(List.of(ctFlussoRiversamento.getIdentificativoFlusso()), true);
+			flowValidatorService.validateOrganization(ctFlussoRiversamento, ingestionFlowFileDTO);
+
+			List<PaymentsReportingDTO> dtoListToSave = paymentsReportingMapperService.mapToDtoList(ctFlussoRiversamento, ingestionFlowFileDTO);
+
+			String iuf = dtoListToSave.get(0).getFlowIdentifierCode();
```

Comment:
let's read it from ctFlussoRiversamento, where it's a single value, reading it as the first from a List could be misunderstood